starlette/endpoints.py

```diff
@@ -10,6 +10,14 @@
 from starlette.types import Message, Receive, Scope, Send
 from starlette.websockets import WebSocket
 
+if False:  # TYPE_CHECKING
```

Comment:
There's a built-in flag in `typing` for this :-)
```suggestion
if typing.TYPE_CHECKING:
```

Reply:
Yes, makes sense (given that this is py36+), thanks!
Not going to update it for now though, given its staleness already.